int16[8] channel_id		# ADC channel IDs, negative for non-existent
float32[8] channel_value	# ADC channel value in volt, valid if channel ID is positive
